context/context_test.go

```diff
@@ -280,3 +280,27 @@ func TestExplicitSubresourcePathLoading(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestSetVariablesFromArguments(t *testing.T) {
+	vars := []string{"version=phony-git-sha"}
+	ctx, _ := LoadContextFromFile("testdata/default-loading.yaml")
+
+	err := ctx.SetVariablesFromArguments(&vars)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if version := ctx.Global["version"]; version != "phony-git-sha" {
+		t.Errorf(`Expected variable "version" to have value "phony-git-sha" but was "%s"`, version)
+	}
+}
+
+func TestSetInvalidVariablesFromArguments(t *testing.T) {
+	vars := []string{"version: phony-git-sha"}
```

Comment:
In some ways tests are a part of a project's documentation - I'd like if the example for passing variables via the CLI used something other than `git-sha` because people may confuse that with the [gitHEAD](https://github.com/tazjin/kontemplate/blob/09869cf8fca09d3b1076e4ee558b8ca9fc91b733/templater/templater.go#L173) template function.

Suggestion: `some-service-version` or something similar.